components/kyma-operator/pkg/steps/installation.go

```diff
@@ -115,22 +128,29 @@ func (steps *InstallationSteps) processComponents(installationData *config.Insta
 
 		steps.PrintStep(stepName)
 
-		processErr := step.Run()
-		if steps.errorHandlers.CheckError("Step error: ", processErr) {
-			_ = steps.statusManager.Error(component.GetReleaseName(), stepName, processErr)
-			return processErr
+		err := retry.Do(
+			func() error {
+				processErr := step.Run()
+				if steps.errorHandlers.CheckError("Step error: ", processErr) {
+					_ = steps.statusManager.Error(component.GetReleaseName(), stepName, processErr)
+					return processErr
```

Comment:
This solution retries failed attempts, but it means `processErr` is not logged at all. If next attempt succeeds, we don't know why the first failed. Please log the error explicitly so it's not "hidden".